scantool/scantool_850/templates/ecu_item.mustache

```diff
@@ -0,0 +1 @@
+    { .addr = 0x{{ecu_address}}, .desc = "{{ecu_description}}", .dtc_prefix = "{{ecu_prefix}}" },
```

Comment:
As I mentioned earlier, unless there's a really compelling reason to use designated initializers, I think `{0x{{ecu_address}}, "{{ecu_description}}", "{{ecu_prefix}}" },` will be unambiguously clear ?